numpy/_core/include/numpy/ndarraytypes.h

```diff
@@ -1514,13 +1514,13 @@ PyArray_STRIDE(const PyArrayObject *arr, int istride)
 }
 
 static inline NPY_RETURNS_BORROWED_REF PyObject *
-PyArray_BASE(PyArrayObject *arr)
+PyArray_BASE(const PyArrayObject *arr)
```

Comment:
Wasn't sure about this one but assumed we don't want users to modify the base through this. 

On a related note I left `PyArray_BYTES` and `PyArray_DATA` as non-const out of these

Reply:
Isn't const fine in all of these cases, since the object is const for the function call?

Reply:
Good point - I'll add those as well